### ISSUE TITLE
Add document Title for accessibility

### ISSUE THREAD

**Issue by jiromaykin** [open-formulieren/open-forms#2491]
### Thema / Theme

Frontend

### Omschrijving / Description

Currently the user only sees "openforms" as a `<head><title>` - for accessibility reasons it would be more helpful to have the organization_name here (which will also become visible in the browser history). In case the admin has not assigned an organization_name, there should be a placeholder.

### Added value / Toegevoegde waarde

This will increase accessibility, more ingo here: https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html
"The descriptive title of an HTML Web page is marked up with the <title> element so that it will be displayed in the title bar of the user agent... **_Titles identify the current location without requiring users to read or interpret page content_**..."

### Aanvullende opmerkingen / Additional context

See also this Accessibility issue about meaningful titles:
https://github.com/open-formulieren/open-forms/issues/2267

**Comment by jiromaykin** [comment]
Note to self: possible solution
`<title>{% block title %}{% if config.organization_name %}{{ config.organization_name }} {% else %}Openforms{% endif %}{% endblock %}</title>`

**Comment by sergei-maertens** [comment]
Fixed via ef401cd0b3ee056444cc9f8d4a909db6b115eee6
